src/index.js

```diff
@@ -169,15 +171,19 @@ class PhoneInput extends React.Component {
     const inputNumber = props.value.replace(/\D/g, '') || '';
 
     let countryGuess;
-    if (inputNumber.length > 1) {
-      // Country detect by phone
-      countryGuess = this.guessSelectedCountry(inputNumber.substring(0, 6), props.country, onlyCountries, hiddenAreaCodes) || 0;
-    } else if (props.country) {
-      // Default country
-      countryGuess = onlyCountries.find(o => o.iso2 == props.country) || 0;
-    } else {
-      // Empty params
+    if (props.disableInitialCountryGuess){
       countryGuess = 0;
+    }else{
```

Comment:
Why adding an additional level of if / else? Can you just add this condition at the beginning of if / else chain?

Reply:
Fixed.